hadoop-hdds/interface-server/src/main/proto/proto.lock

```diff
@@ -457,6 +457,12 @@
                 "name": "storageReport",
                 "type": "StorageReportProto",
                 "is_repeated": true
+              },
+              {
+                "id": 2,
```

Comment:
Yes, because proto.lock file getting updated. It is causing issues. I have opened a Jira but it is said that we should consider updating proto.lock files should be updated only at end of release.